doc_files/iter8_install.md

```diff
@@ -1,52 +1,46 @@
 
-These instructions show you how to set up iter8 on Kubernetes with Istio or with Knative.
+These instructions show you how to set up iter8 on Kubernetes with Istio.
 
 ## Prerequisites
 
 * Kubernetes v1.11 or newer.
-
-If you want to use iter8 with Istio, we require:
-
 * Istio v1.1.5 and newer.
 * Your Istio installation must have at least the **istio-pilot** as well as **telemetry** and **Prometheus** enabled.
 
-If you want to use iter8 with Knative, we require:
-
-* [Knative 0.6](https://knative.dev/docs/install/) or newer.
-
 ## Install iter8 on Kubernetes
 
-### Quick installation
+iter8 has two components, _iter8_analytics_ and _iter8_controller_. To install them, follow these instructions.
+
+### Quick installation (latest release)
 
-iter8 has two components, _iter8_analytics_ and _iter8_controller_. These can be installed for use with Istio as follows:
+To install the latest iter8 release with the default settings, you can apply the default yaml files for _iter8-analytics_ and _iter8-controller_ by running the following command:
 
 ```bash
 kubectl apply \
-    -f https://raw.githubusercontent.com/iter8-tools/iter8-analytics/master/install/kubernetes/iter8-analytics.yaml \
-    -f https://raw.githubusercontent.com/iter8-tools/iter8-controller/master/install/iter8-controller.yaml 
+    -f https://github.com/iter8-tools/iter8-analytics/releases/latest/download/iter8-analytics.yaml \
+    -f https://github.com/iter8-tools/iter8-controller/releases/latest/download/iter8-controller.yaml
 ```
 
-To install for use with knative, modify the first file:
+### Customized installation via Helm charts (latest release)
 
-```bash
-kubectl apply \
-    -f https://raw.githubusercontent.com/iter8-tools/iter8-analytics/master/install/knative/iter8-analytics.yaml \
-    -f https://raw.githubusercontent.com/iter8-tools/iter8-controller/master/install/iter8-controller.yaml 
-```
+In case you need to customize the installation of iter8's latest release, use the Helm charts listed below:
 
-### Customized installation
+* _iter8-analytics_: [https://github.com/iter8-tools/iter8-analytics/releases/latest/download/iter8-analytics-helm-chart.tar](https://github.com/iter8-tools/iter8-analytics/releases/latest/download/iter8-analytics-helm-chart.tar)
 
-In case you need to change the default configuration options used in the quick installation, use the helm charts directly by cloning the projects:
+* _iter8-controller_: [https://github.com/iter8-tools/iter8-controller/releases/latest/download/iter8-controller-helm-chart.tar](https://github.com/iter8-tools/iter8-controller/releases/latest/download/iter8-controller-helm-chart.tar)
+
+### Installing an older release
+
+In case you need to install an old iter8 release, please, refer to its corresponding documentation in the link below. In the URL below, you need to replace the string `<release>` with the string corresponding to your desired release. For example, a valid release string would be `v0.0.1`.
```

Comment:
nip pick. remove the comma after "please"

Reply:
Done.